engine/apps/mattermost/views.py

```diff
@@ -61,3 +62,29 @@ def perform_destroy(self, instance):
             channel_id=instance.channel_id,
         )
         instance.delete()
+
+
+class MattermostEventView(APIView):
+    def get(self, request, format=None):
```

Comment:
This event requests will be authenticated using the signing secret and jwt, right?

Reply:
Yup the secret that is sent as part of the message !